src/PKSim.Presentation/Presenters/Observers/ImportObserversPresenter.cs

```diff
@@ -0,0 +1,128 @@
+using OSPSuite.Presentation.Presenters;
+using OSPSuite.Utility;
+using OSPSuite.Utility.Exceptions;
+using OSPSuite.Utility.Extensions;
+using PKSim.Assets;
+using PKSim.Core.Model;
+using PKSim.Core.Services;
+using PKSim.Presentation.DTO.Observers;
+using PKSim.Presentation.Views.Observers;
+
+namespace PKSim.Presentation.Presenters.Observers
+{
+   public interface IImportObserversPresenter : IObserverItemPresenter
+   {
+      IEnumerable<IObserverBuilder> Observers { get; }
+      void RemoveObserver(ImportObserverDTO observer);
+      void AddObserver();
+      void SelectObserver(ImportObserverDTO observer);
+      bool ShowFilePath { get; set; }
+   }
+
+   public class ImportObserversPresenter : AbstractSubPresenter<IImportObserversView, IImportObserversPresenter>, IImportObserversPresenter, ILatchable
+   {
+      private readonly IObserverInfoPresenter _observerInfoPresenter;
+      private readonly IDialogCreator _dialogCreator;
+      private readonly IObserverTask _observerTask;
+      private readonly List<ImportObserverDTO> _observerDTOs = new List<ImportObserverDTO>();
+      private PKSimObserverBuildingBlock _observerBuildingBlock;
+      public bool IsLatched { get; set; }
+
+      public ImportObserversPresenter(
+         IImportObserversView view,
+         IObserverInfoPresenter observerInfoPresenter,
+         IDialogCreator dialogCreator,
+         IObserverTask observerTask) : base(view)
+      {
+         _observerInfoPresenter = observerInfoPresenter;
+         _dialogCreator = dialogCreator;
+         _observerTask = observerTask;
+         AddSubPresenters(_observerInfoPresenter);
+         _view.AddObserverView(_observerInfoPresenter.View);
+         _view.BindTo(_observerDTOs);
+      }
+
+      public IEnumerable<IObserverBuilder> Observers { get; } = new List<IObserverBuilder>();
+
+      public void Edit(PKSimObserverBuildingBlock observerBuildingBlock)
+      {
+         //This is to prevent edit triggered by action happening in this presenter
+         if (IsLatched)
+            return;
+
+         _observerBuildingBlock = observerBuildingBlock;
+         _observerDTOs.Clear();
+         _observerBuildingBlock.Observers.Each(x => addObserver(x));
+         _view.Rebind();
+      }
+
+      public void RemoveObserver(ImportObserverDTO observerDTO)
+      {
+         addCommand(() => _observerTask.RemoveObserver(observerDTO.Observer, _observerBuildingBlock));
+         _observerDTOs.Remove(observerDTO);
+         updateView();
+      }
+
+      public void AddObserver()
+      {
+         var newFile = getNewFile();
+         if (string.IsNullOrEmpty(newFile))
+            return;
+
+         var observer = _observerTask.LoadObserverFrom(newFile);
+         if (observer == null)
+            return;
+
+         if (_observerDTOs.ExistsByName(observer.Name))
+            throw new OSPSuiteException(PKSimConstants.Error.NameAlreadyExistsInContainerType(observer.Name, ObjectTypes.ObserverBuildingBlock));
```

Comment:
This is not consistent with Load-Behaviour of other objects (templates, MoBi-Simulations from pkml etc.)
In other cases a Rename-Dialog is shown and user can rename loaded object

Reply:
Yeah but this is not the same level however. We are loading an element IN a building block which we never do in PKSim. 
If yo u feel strongly about it, please create a specific issue. I think this is overkill in this case, especially since the observer are readonly in pksim